bin/check_samplesheet.py

```diff
@@ -146,7 +146,7 @@ def sniff_format(handle):
     peek = read_head(handle)
```

Comment:
again, forgot to have prettier ignore this I believe, and this will also be modified in a future PR from the nf-core check_samplesheet because I like the solution Mert applied to phylorthology and that's what I have in hifi2genome as well